### ISSUE TITLE
Template dashboard doesn't show repeating rows in report

### ISSUE THREAD

**Issue by jahluwalia** [IzakMarais/reporter#79]
Hi,

I've got the following dashboard:
https://grafana.com/dashboards/2587

When i generate a report, I only see two rows being rendered in the report. The last row http_req_blocked isn't in the report at all. 

Any ideas?

thanks,
jas

**Comment by jahluwalia** [comment]
Does reporter support grafana templating? That's the only thing I can think of where the templating in the dasbhoard.json isn't working.

**Comment by IzakMarais** [comment]
Yes interaction between templating and the grafana dashboards HTTP API is still a bit of a mystery. 
I think [this](https://github.com/IzakMarais/reporter/issues/55#issuecomment-382741575) comment explains it.

**Comment by IzakMarais** [comment]
In #75 @tomasbastien got a good answers by posting a question on the Grafana community page. Perhaps the interaction between template dashboards and the API is a topic for a question there.

**Comment by aleguerra05** [comment]
Hello @IzakMarais 
I'm really interested in this functionality. Maybe I could send you some pull requests on this. Would you be able to test them and accept the right one? Please, we will really appreciate it.
You can write to me at gmail: aleguerra05
Thank you

**Comment by IzakMarais** [comment]
Hi, 

I first need to get to accepting the currently open good PRs. I’m struggling finding time to support this project. ‘

Regards
Izak
